apps/classroomio-com/src/routes/bootcamps/+page.svelte

```diff
@@ -0,0 +1,257 @@
+    },
+    {
+      title: 'Customizable Experience',
+      description: "Tailor the learning experience to your company's specific needs and branding."
+    },
+    {
+      title: 'Data-Driven Insights',
+      description: 'Gain valuable insights into employee performance and learning patterns.'
+    }
+  ];
+</script>
+
+<svelte:head>
+  <title>Blog | ClassroomIO</title>
+</svelte:head>
+
+<section>
+  <PageHeader
+    className="flex flex-col lg:items-center items-start lg:justify-center justify-start gap-y-6 lg:px-28 px-8"
+  >
+    <div class="bg-[#DCE5FF] py-2 px-4 rounded-full border-[#C2D2FF] border-2 text-center">
+      <p class=" text-sm md:text-base text-[#4B5563] text-center font-medium">
+        Solution for Bootcamps
+      </p>
+    </div>
+    <div
+      class="w-full flex flex-col lg:items-center items-start lg:justify-center justify-start gap-y-8"
+    >
+      <div
+        class="w-full flex flex-col lg:items-center items-start lg:justify-center justify-start gap-y-4"
+      >
+        <h1
+          class="w-[100%] mx-auto m-0 text-4xl lg:text-7xl md:text-6xl font-bold text-slate-900 lg:text-center"
+        >
+          Elevate Your <span class="text-blue-700 relative"> Bootcamps</span> with
+          <span class="text-blue-700 relative"> Innovative</span> Tools
+        </h1>
+        <p class="w-[100%] lg:w-[60%] lg:text-center font-normal text-lg text-slate-700">
+          From intelligent course creation to student collaboration tools, ClassroomIO provides
+          everything you need to run a cutting-edge, high-impact bootcamp.
+        </p>
+      </div>
+      <button
+        class="py-5 px-6 bg-[#1D4ED8] border-transparent outline-transparent rounded-lg text-white text-lg font-semibold"
+        >Get Started for Free 🚀</button
+      >
+    </div>
+  </PageHeader>
+
+  <section
+    class="w-full lg:py-24 py-16 lg:px-28 px-8 flex lg:flex-row flex-col lg:items-center lg:justify-between justify-center gap-x-6 gap-y-8"
+  >
+    <div class="w-full lg:w-[40%] gap-y-4 flex flex-col items-start justify-start">
+      <div class="w-fit py-2 px-4 rounded-full border-[#C2D2FF] border-2 text-center">
+        <p class=" text-sm md:text-base text-[#4B5563] text-center font-medium">Amazing benefits</p>
+      </div>
+
+      <div class="md:w-full flex flex-col items-start justify-start gap-y-4">
+        <h1 class="md:text-5xl text-3xl font-medium text-[#282828]">
+          Unlock Your Team's Potential and Drive Success
+        </h1>
+        <p class="text-[#656565] text-xl font-light">
+          Experience the power of collaborative learning with ClassroomIO. Engage your workforce
+          with personalized training that aligns with your company's goals.
+        </p>
+      </div>
+    </div>
+    <div class="w-full flex flex-col lg:w-[50%] lg:grid lg:grid-cols-2 gap-y-4 lg:gap-4">
+      <div
+        class="w-full md:w-[100%] md:p-6 p-4 border border-[#D9E0F5] rounded-2xl flex flex-col gap-y-6"
```

Comment:
delete `md:w-[100%]`, it's tautology setting a full width for small and medium screens when you can just set it once and it's applied to both.